src/worker/jobs/chainSubscribers.ts

```diff
@@ -147,7 +147,7 @@ class ChainSubscribers {
         const parsedLog = this.ifaceERC20.parseLog(log);
         let result: ethers.utils.LogDescription | undefined = undefined;
         // only transactions to community contracts (donations)
```

Comment:
change the comment to `// only transactions to community contracts (donations) or DAO`

Reply:
done!